pkg/api/client.go

```diff
@@ -54,6 +55,14 @@ type httpClient struct {
 	timeout    time.Duration
 }
 
+type ConfigAgent struct {
+	EnvironmentName string
+	IsDocker        bool
+	AgentName       string
+}
+
+var CfgAgent ConfigAgent
```

Comment:
if CfgAgent accessed anywhere else except for this package?

Reply:
he is creating it in the agent package

Reply:
i was just thinking of scope with upper case CfgAgent v. cfgAgent

Reply:
oops... sorry :)...

Reply:
2 comments > 1